packages/fiori/src/Timeline.js

```diff
@@ -115,7 +127,7 @@ class Timeline extends UI5Element {
 	}
 
 	get ariaLabel() {
-		return Timeline.i18nBundle.getText(TIMELINE_ARIA_LABEL);
+		return getEffectiveAriaLabelText(this);
```

Comment:
Don't we need the TIMELINE_ARIA_LABEL as default in case accessibleName is not set?
If we don't - please explain in the commit message.

Reply:
This text results to "Timeline", which describes the element and could later on be moved into aria-roledescription attribute as documented: https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2382053542
I think we need this label alongside the <code>accessibleName</code> text. Also will be good to update the upper status page.

Reply:
The only thing to do is:

```js
get ariaLabel() {
  return this.accessibleName || Timeline.i18nBundle.getText(TIMELINE_ARIA_LABEL);
}